hpOneView/image_streamer/resources/artifact_bundles.py

```diff
@@ -24,92 +24,25 @@
 standard_library.install_aliases()
 
 
-from hpOneView.resources.resource import ResourceClient, extract_id_from_uri
+from hpOneView.resources.resource import Resource, ResourceFileHandlerMixin, extract_id_from_uri
 
 
-class ArtifactBundles(object):
+class ArtifactBundles(ResourceFileHandlerMixin, Resource):
 
     URI = '/rest/artifact-bundles'
-    DEPLOYMENT_GROUPS_URI = '/rest/deployment-groups/'
     BACKUPS_PATH = '/rest/artifact-bundles/backups'
     BACKUP_ARCHIVE_PATH = '/rest/artifact-bundles/backups/archive'
-    STOP_CREATION_PATH = '/stopArtifactCreate'
     DOWNLOAD_PATH = '/rest/artifact-bundles/download'
 
     DEFAULT_VALUES = {
-        '300': {"type": "ArtifactsBundle"}
+        '800': {"type": "ArtifactsBundle"},
+        '1000': {"type": "ArtifactsBundle"},
+        '1020': {"type": "ArtifactsBundle"},
+        '1600': {"type": "ArtifactsBundle"}
```

Comment:
you can define this type in init() as they are same for all API versions.

Reply:
done